### DISCUSSION TITLE
Timeout adjustments

### DISCUSSION THREAD

**Discussion by Xel-Naha** [bpg/terraform-provider-proxmox#1037] [Ideas]
hi,

I ran into an issue today whereby my container took a while to shutdown before destruction and this ran into terraform timeout. Reading the documentation on terraform it mentions a configurable timeout settings in the resource part:

https://developer.hashicorp.com/terraform/language/resources/syntax#operation-timeouts

however I Believe this provider has no such options does it? How hard would it be to add these as it will certainly help with some longer running actions.

thanks

**Comment by bpg-dev** [comment]
There timeout arguments defined for the container resource, like [`timeout_create`](https://registry.terraform.io/providers/bpg/proxmox/latest/docs/resources/virtual_environment_container#timeout_create), etc.
